app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ReviewsController < ApplicationController
+  def new
+    @review = Review.new
+    @beach = Beach.find(params[:beach_id])
+  end
+
+  def create
+    @user = User.first
```

Comment:
Do you mean `current_user` here? Basically you want the user who is the current user of the site to be the user associated with a new review.

```ruby
@review = Review.new(review_params)
@beach = Beach.find(params[:beach_id])
@review.beach = @beach
@review.user = current_user
```